HelpSource/Classes/Psubdivide.schelp

```diff
@@ -0,0 +1,81 @@
+class:: Psubdivide
+summary:: partition a value into n equal subdivisions
+related:: Classes/Pdup
+categories:: Streams-Patterns-Events>Patterns>Repetition
+
+description::
+
+A filter pattern designed for a stream of float durations.
+
+Subdivides each duration by each stutter and yields that value stutter times. A stutter of 0 will skip the duration value, a stutter of 1 yields the duration value unaffected.
```

Comment:
Do we want to use the work `stutter` here?

Reply:
Oops, that's definitely a copy-paste error - thanks for catching!